tests/clone_test.go

```diff
@@ -874,3 +729,148 @@ func createSourceVM(options ...libvmi.Option) (*virtv1.VirtualMachine, error) {
 	virtClient := kubevirt.Client()
 	return virtClient.VirtualMachine(vm.Namespace).Create(context.Background(), vm, v1.CreateOptions{})
 }
+
+func createSnapshot(vm *virtv1.VirtualMachine) *snapshotv1.VirtualMachineSnapshot {
```

Comment:
Consider passing vm.Name and vm.Namespace instead of the full VM if it is not required otherwise.

Reply:
DONE